### PR TITLE
[ADD] auth_cas 9.0 ver.

### PR DESCRIPTION
**Pull request by etoanik**
add auth_cas 9.0 version from 10.0

### CODE REVIEW

**Comment by elicoidal** [comment]
@etoanik Thanks for the contribution!
Let me get somebody to review it so that we can merge it soon.
@JoJoJoJoJoJoJo @joycehuan @victormartinelicocorp

File: auth_cas/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Roméo Guillot Roméo Guillot (http://www.opensource-elanz.fr).
+# © 2016-2018 Elico Corp (https://www.elico-corp.com).
```
Comment (elicoidal) [inline auth_cas/__openerp__.py:3]: 2016-2019 (everywhere)

**Review by elicoidal — CHANGES_REQUESTED** [review]
LGTM. Not tested (did you?)

**Comment by etoanik** [comment]
I had install in my odoo 9.0 env.

File: auth_cas/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+# © 2016-2018 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import urllib
+from openerp.addons.web.controllers.main import login_and_redirect
+from openid.cryptutil import randomString
+from openerp.exceptions import AccessDenied
+import urlparse
+
+from ..pycas import login
```
Comment (martintorresvm) [inline auth_cas/controllers/main.py:12]: We should not have `pycas` as an embed library, we should add a fixed and stable version pip install on Travis and add this as external dependency named on __opernerp__.py

See OCA guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#17external-dependencies

See project https://pypi.org/project/pycas/

See an example https://github.com/OCA/reporting-engine/blob/11.0/report_xlsx/__manifest__.py#L16

**Review by victor-ditay — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
I will merge for the time being. Later on we will remove the dependency
Thanks for the contribution!

**Comment by elicoidal** [comment]
@etoanik Thanks for the contribution!

**Pull request merged**